### ISSUE TITLE
Support for Firefox?

### ISSUE THREAD

**Issue by Tomatoide** [SpiritCroc/DefaultDarkTheme-oms#29]
Is it possible? thanks

**Comment by SpiritCroc** [comment]
Most likely won't happen, theming browsers doesn't make much sense to me since most content you'll see is websites, which won't get touched by OMS themes.

**Comment by Tomatoide** [comment]
Yes but in the case of firefox Dark Reader is a very popular extension that is basically dark mode for websites, which leaves the browser itself as the eye sore.

**Comment by Tomatoide** [comment]
This could be a really nice workaround for this long highly requested feature in firefox for android (dark theme)

**Comment by SpiritCroc** [comment]
As I said, this is nothing what OMS/Substratum themes can do.

OMS/Substratum can overlay (= theme) the following:
- Android app resources

What you want to change:
- Website style sheet

Remember that this is an Android Substratum theme, not a firefox extension.

**Comment by Tomatoide** [comment]
I understand, you got me wrong, I know this is a substratum theme, and yes what I'm requesting is a dark theme for org.mozilla.firefox, the android app resources as you said, may be something similar to this:

![](https://i.redd.it/lu75zfygoui01.jpg)

**Comment by SpiritCroc** [comment]
I see. My first point still stands.
Judging from your screenshot, you already found a way to make it dark either way?

**Comment by Tomatoide** [comment]
No, that's from a substratum theme that works on android 9..
I disagree with your first point though..
When you have a dark themed phone ui and dark themed websites / content, and the browser ui being the exception with its bright white ui, it becomes very painful..
First when you open the browser and get blasted by the bright white, then as you are browsing with browser elements such as the address bar.. I mean there is a reason why dark theme in web browsers is a highly requested feature nowadays

**Comment by SpiritCroc** [comment]
You don't give up easily until you get an answer you like, do you?

No need to convince me that dark applications look better than light ones, I mean have a guess why this theme exists...
Also, it doesn't matter much to me how "highly requested" this is, I'm not getting any profit out of doing this either way.

That said, I might do it at some point since I use firefox myself, but my TODO list is rather long, and this is not a high priority to me.
Thanks for understanding.

**Comment by Tomatoide** [comment]
Regardless of the unneeded passive aggressiveness, I appreciate considering this in the future, thanks.
